CLI/src/CLI.cs

```diff
@@ -129,6 +129,21 @@ private ReturnCode Compile()
 
             //Normalize TypeCobolOptions, the parser does not need to go beyond SemanticCheck for the first phase
             var typeCobolOptions = new TypeCobolOptions(_configuration);
+            ReturnCode optionRetCode = typeCobolOptions.OptionStatusCode;
+            switch (optionRetCode)
+            {
+                case ReturnCode.Success:
+                    break;
+#if EUROINFO_RULES
+                case ReturnCode.CopyNameMapFileError:
+                    //Exit only if we fail to read a Cpy Name file other than the default one.
+                    if (_configuration.CpyCopyNamesMapFilePath.Equals(TypeCobolConfiguration.DefaultLogFileName))
+                        return optionRetCode;
```

Comment:
Is it required ? If an error occured while loading the file it will be logged, but still we can let the process continue. This would allow us to delete the `OptionStatusCode` property and associated checking.
Besides, the comment is misleading: here we stop only if the file path is the default one.
And I don't see why we should handle the error on the default path differently than on a user-supplied path.

Reply:
So let be sure now that we always have a default file present, otherwise we could never be able to tests or run CLI without that file. We still have to do it internally using our TFS build process.